packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -260,9 +280,23 @@ export class ListItem
 
   /**
    * Fires when the drag handle is selected.
+   *
+   * @deprecated use calciteListItemSortHandleOpen instead.
    */
   @Event({ cancelable: false }) calciteListItemDragHandleChange: EventEmitter<void>;
```

Comment:
can I just remove this since its a breaking change already?

Reply:
No, I'd like to keep this PR focused.

Reply:
Apologies, I thought you were suggesting removing previously deprecated members. If the "keyboard list move" changes made the prop and event irrelevant, they should be fine to remove. Let’s just make sure to note this in the breaking change footer and our 3.x breaking change doc.